include/uv-win.h

```diff
@@ -45,8 +45,8 @@ typedef struct pollfd {
 
 #ifndef _malloca
 # if defined(_DEBUG)
-#  define _malloca(size) malloc(size)
-#  define _freea(ptr) free(ptr)
+#  define _malloca(size) uv__malloc(size)
+#  define _freea(ptr) uv__free(ptr)
```

Comment:
FWIW I'm +1 to always doing a heap allocation.
On Feb 27, 2015 7:00 PM, "Ben Noordhuis" <notifications@github.com> wrote:

> In include/uv-win.h
> <https://github.com/libuv/libuv/pull/231#discussion_r25524815>:
>
> > @@ -45,8 +45,8 @@ typedef struct pollfd {
> >
> >  #ifndef _malloca
> >  # if defined(_DEBUG)
> > -#  define _malloca(size) malloc(size)
> > -#  define _freea(ptr) free(ptr)
> > +#  define _malloca(size) uv__malloc(size)
> > +#  define _freea(ptr) uv__free(ptr)
>
> I agree, I think it can be scrapped. It'd be nice to have confirmation
> from @piscisaureus <https://github.com/piscisaureus> though.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/libuv/libuv/pull/231/files#r25524815>.
>